Makefile.source.mjs

```diff
@@ -468,9 +469,26 @@ target["new-version"] = function () {
   yarn(["release-tool", "version", "-f", "@babel/standalone"]);
 };
 
-target["new-version"] = function () {
-  target["new-version-checklist"]();
+target["new-babel-8-version-prepare"] = function () {
+  shell.exec("git pull --rebase");
 
-  exec("git", ["pull", "--rebase"]);
-  yarn(["release-tool", "version", "-f", "@babel/standalone"]);
+  const pkg = JSON.parse(readFileSync("./package.json", "utf8"));
+  const nextVersion = semver.inc(pkg.version_babel8, "prerelease");
+  pkg.version_babel8 = nextVersion;
+  writeFileSync("./package.json", JSON.stringify(pkg, null, 2) + "\n");
+  shell.exec("git add package.json");
+  shell.exec("git commit -m 'Bump Babel 8 version to " + nextVersion + "'");
```

Comment:
Nit: We can use `exec("git", ["commit", ... ])` so we don't have to take care of quotes required for commit message.

Reply:
The reason I used `shell.exec` is because manually writing the quotes is less annoying than having to quote every single part of the command to put it in an array 😅

Reply:
You can try copilot, it can help you do this very well. :)
Also `exec()` will default to `inheritStdio` and check the return code.

Reply:
Done!